content/en/docs/examples/virtual-machines/multi-network/index.md

```diff
@@ -75,22 +68,31 @@ The following example updates the `/etc/hosts` file with the {{< gloss >}}Istiod
    /etc/hosts is an easy to use example. It is also possible to use a real DNS and certificate for Istiod, this is beyond
    the scope of this document.
 
+2. Obtain `tcp-istiod` port for Istio ingress gateway and export it as `CAPORT`. Add it into `/var/lib/istio/envoy/sidecar.env`.  Revisit the [preparing the cluster](#preparing-the-kubernetes-cluster-for-vms) section to learn how to obtain the `tcp-istiod` port.
```

Comment:
```suggestion
1. Obtain `tcp-istiod` port for Istio ingress gateway and export it as `CAPORT`. Add it into `/var/lib/istio/envoy/sidecar.env`.  Revisit the [preparing the cluster](#preparing-the-kubernetes-cluster-for-vms) section to learn how to obtain the `tcp-istiod` port.
```

Reply:
> Revisit the [preparing the cluster](#preparing-the-kubernetes-cluster-for-vms) section to learn how to obtain the `tcp-istiod` port.

I don't see any instructions about obtaining `tcp-istiod` port at that link.

Reply:
@frankbu the two :$CAPORT are referencing `tcp-istiod` port
   {{< text bash >}}
    $ echo "CA_ADDR=istiod.istio-system.svc:$CAPORT" >> cluster.env
    $ echo "ISTIO_PILOT_PORT=$CAPORT >> cluster.env
    {{< /text >}}

Reply:
@irisdingbj Yes, I see that, but there is no instructions for how to set CAPORT, even though it says "Revisit the preparing the cluster section ..." to learn how to.